moduledata.go

```diff
@@ -24,10 +24,11 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
-	"golang.org/x/mod/semver"
 	"io"
 	"strconv"
 	"strings"
+
+	"golang.org/x/mod/semver"
```

Comment:
Could you keep the legacy style here? This can prevent the possible conflict with #85

Reply:
`gofumpt` did it, I suppose I can keep the original form.